unpackaged/pre/acc_record_types/objects/Account.object

```diff
@@ -3,7 +3,7 @@
     <recordTypes>
         <fullName>Academic_Program</fullName>
         <active>true</active>
-        <description>An Academic Program offered at the University.</description>
```

Comment:
@juliesnow  Please note, I've updated the description of the Academic Program and University Department record types on the Account to use the term "educational institution" instead of university.